web/docs/tql2/operators/from_file.md

```diff
@@ -0,0 +1,69 @@
+# from_file
+
+:::warning
+This operator is still under active development.
+:::
+
+Reads one or multiple files using from a filesystem.
+
+```tql
+from_file url:string, [watch=bool, remove=bool, path_field=field { … }]
+```
+
+## Description
+
+### `url: string`
+
+URL or local filesystem path where data should be read from.
+
+The characters `*` and `**` have a special meaning. `*` matches everything
+except `/`. `**` matches everything including `/`. The sequence `/**/` can also
+match nothing. For example, `foo/**/bar` matches `foo/bar`.
+
+### `watch = bool (optional)`
+
+In addition to processing all existing files, this option keeps the operator
+running, watching for new files that also match the given URL.
+
+Defaults to `false`.
```

Comment:
The interval should be mentioned here, and also what cost this can incur (2x LIST per interval + 1x READ per new file).

Reply:
We agreed to just document the current watch interval.